api/ai/interfaces/algorithm_options.py

```diff
@@ -128,8 +128,8 @@ def get_schema() -> Schema:
                 SchemaOptional("enemy_behaviour"): Or(
                     *[behaviour.value for behaviour in RelationshipBehaviour]
                 ),
-                SchemaOptional("attributes_to_diversify"): List[int],
-                SchemaOptional("attributes_to_concentrate"): List[int],
+                SchemaOptional("attributes_to_diversify"): Or(list[int], []),
```

Comment:
seems weird to me that the lower case is needed, and that we need to explicitly union set with it just to account for empty lists...

I'll trust you if you say so but seems odd